security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -80,6 +84,13 @@ const (
 	caCertID = "ca-cert.pem"
 	// caPrivateKeyID is the private key file of CA.
 	caPrivateKeyID = "ca-key.pem"
+
+	// The name of the ConfigMap in each namespace storing the CA cert of self-signed CA.
+	CACertNamespaceConfigMap = "istio-ca-ns-configmap"
+	// The data name in the ConfigMap of each namespace storing the CA cert of self-signed CA.
+	CACertNamespaceConfigMapDataName = "ca-cert-ns.pem"
+	CACertNamespaceInsertInterval    = time.Second
+	CACertNamespaceInsertTimeout     = time.Second * 2
```

Comment:
Why so small? What happens if we have a namespace created, there is some issue, then are we permanently broken?

Reply:
Every namespaceResyncPeriod, namespaceUpdated() will be invoked for every namespace. So if there is some issue updating CA certificate in a namespace, the update will be retried next time.

Reply:
We are writing this configmap every 5s to every namespace? That is likely to cause issues, that seems way to frequently

Reply:
Based on offline discussion, namespaceResyncPeriod is changed from 5 seconds to 30 seconds.